backend/globaleaks/jobs/delivery.py

```diff
@@ -91,21 +94,30 @@ def write_plaintext_file(sf, dest_path):
         log.err("Unable to create plaintext file %s: %s", dest_path, excep)
 
 
-def write_encrypted_file(key, sf, dest_path):
+def write_encrypted_file(session, key, sf, dest_path):
+    url_clam_av = ConfigFactory(session, 1).get_val('url_file_analysis')
+    af = FileAnalysis(url = url_clam_av)
```

Comment:
TODO: this code should be refactored as it makes a syncronous call to the antivirus before saving the file and waits for the result to proceed.

In fact priority for a file upload of sensitive information is to ensure that the file is saved.
Other checks on the files like for example a check with a remote file should happen asyncronously.

This processing should happen